packages/integrations/mdx/src/plugins.ts

```diff
@@ -161,9 +161,8 @@ export async function getRemarkPlugins(
 	remarkPlugins = [...remarkPlugins, ...ignoreStringPlugins(mdxOptions.remarkPlugins)];
 
 	// Apply last in case user plugins resolve relative image paths
-	if (config.experimental.contentCollections) {
-		remarkPlugins.push(toRemarkContentRelImageError(config));
-	}
+	remarkPlugins.push(toRemarkContentRelImageError(config));
+
```

Comment:
Don't we still need this flag? Relative assets should only be disallowed in the `src/content/` directory?

Reply:
Hm, `remarkContentRelImageError` checks if a file's in the `contentDir` internally. Shouldn't need a flag before applying (see https://github.com/withastro/astro/blob/main/packages/markdown/remark/src/remark-content-rel-image-error.ts#L13)